nucypher/characters/control/specifications/fields/treasuremap.py

```diff
@@ -0,0 +1,30 @@
+from marshmallow import fields
+from base64 import b64decode, b64encode
+from bytestring_splitter import BytestringSplitter, VariableLengthBytestring
+from nucypher.characters.control.specifications.fields.base import BaseField
+from nucypher.crypto.signing import Signature
+from nucypher.crypto.constants import KECCAK_DIGEST_LENGTH
+from nucypher.crypto.kits import UmbralMessageKit
+
+
+class TreasureMap(BaseField, fields.Field):
+
+    def _serialize(self, value, attr, obj, **kwargs):
+        return b64encode(bytes(value)).decode()
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        return b64decode(value)
+
+    def _validate(self, value):
```

Comment:
Can validate here simply be (?)

```python
try:
    treasure_map = TreasureMap.from_bytes(value):
    return True
except:
    return False
```

I'm unsure whether the `verify` kwarg for `from_bytes` can remain as `True` (default) or needs to be set to `False`.

Reply:
Yes, I'd love it if that worked but unfortunately importing `nucypher.policy.collections.Treasuremap` causes a circular import since Bob is imported into that same file... which I think should maybe be a TODO but maybe there is some reason I'm not aware of.